src/routes.php

```diff
@@ -8,4 +10,14 @@
     Route::get('/pages/{slug}/edit', '\Kirschbaum\LaravelSparkPages\PageController@edit');
 });
 
-Route::get('/{slug1}/{slug2?}', '\Kirschbaum\LaravelSparkPages\PageController@show');
+Route::group(['middleware' => ['web']], function () {
+    try {
+        $pages = Page::select(['slug'])->get();
+
+        foreach ($pages->pluck('slug') as $slug) {

```

Comment:
It runs through all pages and add their route to the `RouteCollection`. However, I think there might be a function to add multiple routes at once.